test/couch_mrview_all_docs_tests.erl

```diff
@@ -107,6 +107,21 @@ should_query_with_limit_and_skip(Db) ->
     ]},
     ?_assertEqual(Expect, Result).
 
+should_query_with_config_limit_and_skip(Db) ->
```

Comment:
I will fix the tests. I agree with @kxepal : it should be ```default_query_limit``` across the board. if there are no objections, I will go ahead and use that name